frontend/src/scenes/insights/insightSceneLogic.tsx

```diff
@@ -190,7 +205,7 @@ export const insightSceneLogic = kea<insightSceneLogicType>([
         '/insights/:shortId(/:mode)(/:subscriptionId)': (
             { shortId, mode, subscriptionId }, // url params
             { dashboard, ...searchParams }, // search params
-            { filters: _filters, q }, // hash params
+            { insight: insightType, q }, // hash params
```

Comment:
why do we want to support hash params and not just search params?

Reply:
no reason in particular here - just taking over the existing system. hash params are quite common for client-side state and afaik there's a higher limit for the number of characters.